js/src/picc.js

```diff
@@ -662,12 +662,10 @@ picc.access.partTimeShare = picc.access.composed(
 );
 
 // only use full-time retention rate, branching on year designation
-// (`retetion
-picc.access.retentionRate = picc.access.composed(
-  picc.fields.RETENTION_RATE,
-  picc.access.yearDesignation,
-  'full_time'
-);
+picc.access.retentionRate = function(d) {
+  var retention = picc.access(picc.fields.RETENTION_RATE)(d);
+  return retention.four_year.full_time || retention.lt_four_year.full_time;
```

Comment:
Identifier 'four_year' is not in camel case.<br>Identifier 'full_time' is not in camel case.<br>Identifier 'lt_four_year' is not in camel case.